hca_ingest/downloader/schema_url.py

```diff
@@ -11,4 +11,8 @@ def concrete_type(self):
 
     @property
     def domain_type(self):
-        return self.url.split('/')[4] if self.url else ''
+        try:
+            domain = ACCEPTED_DOMAINS[[domain in self.url for domain in ACCEPTED_DOMAINS].index(True)]
```

Comment:
It would be clear to refactor this to a function with a meaningful name that captures your intent.